src/main/java/games/strategy/triplea/ai/proAI/ProTechAI.java

```diff
@@ -179,7 +172,7 @@ private static float getStrengthOfPotentialAttackers(final Territory location, f
       }
       final List<Unit> attackPlanes =
           findPlaneAttackersThatCanLand(location, maxFighterDistance, enemyPlayer, data, ignoreTerr, checked);
-      airStrength += allairstrength(attackPlanes, true);
+      final float airStrength = allairstrength(attackPlanes, true);
```

Comment:
This is the first usage of `airStrength` after it was declared above, so I first moved the `airStrength` declaration immediately before this line:

```java
float airStrength = 0.0F;
airStrength += allairstrength(attackPlanes, true);
```

From there, it was apparent that these two lines could simply be merged into a single declaration and initialization.

As there are no further modifications to `airStrength` in this method, it is able to be marked `final`.